pkg/operator/ceph/predicate.go

```diff
@@ -91,6 +92,17 @@ func predicateController(ctx context.Context, client client.Client) predicate.Fu
 						return false
 					}
 				}
+			} else if old, ok := e.ObjectOld.(*api.Certificate); ok {
+				if new, ok := e.ObjectNew.(*api.Certificate); ok {
+					if old.Name == certificateName && new.Name == certificateName {
+						logger.Debugf("webhook certificate %s created reloading the manager to enable the webhook server", certificateName)
+						controller.ReloadManager()
+
+						// No need to ask for reconciliation since the context is going to be terminated when
+						// the signal is caught and the reconcile will run when the controller starts.
+						return false
```

Comment:
How will returning false in this case change the behavior? The default below is already to return false. Is there somewhere else that will trigger a reconcile?

Reply:
I saw above `controller.ReloadManager()` which will cancel the context and it makes reconcile the controller. I followed the same. I had a question about the difference between using `true` and canceling the context.

Reply:
Not sure I follow, let's discuss